src/core/metadata/qgslayermetadataformatter.h

```diff
@@ -30,6 +30,8 @@
  */
 class CORE_EXPORT QgsLayerMetadataFormatter
 {
+    Q_DECLARE_TR_FUNCTIONS()
```

Comment:
Better add `QgsLayerMetadataFormatter` this as context, it can sometimes help to know the context.

Reply:
Fixed